### PR TITLE
Gernate milvus.yaml by code

### PR DESCRIPTION
**Pull request by jiaoew1991**
Signed-off-by: Enwei Jiao <enwei.jiao@zilliz.com>
/kind improvement

### CODE REVIEW

File: cmd/tools/config/generate.go [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+		result = append(result, d)
+	}
+	return result
+}
+
+func collectRecursive(params *paramtable.ComponentParam, data *[]DocContent, val *reflect.Value) {
+	if val.Kind() != reflect.Struct {
+		return
+	}
+	log.Debug("enter", zap.Any("variable", val.String()))
```
Comment (congqixia) [inline cmd/tools/config/generate.go:54]: zap.String

**Review by congqixia — COMMENTED** [review]
/lgtm
/cc @xiaofan-luan @Bennu-Li

**Comment by jiaoew1991** [comment]
rerun ut

**Comment by haorenfsa** [comment]
part of: https://github.com/milvus-io/milvus/issues/21856

**Comment by jiaoew1991** [comment]
rerun ut

File: configs/milvus.yaml [hunk 2]
```diff
@@ -16,424 +16,381 @@
...
     path: stdout
-    level: info # Only supports debug, info, warn, error, panic, or fatal. Default 'info'.
-  use:
-    # please adjust in embedded Milvus: true
-    embed: false # Whether to enable embedded Etcd (an in-process EtcdServer).
-  data:
-    # Embedded Etcd only.
-    # please adjust in embedded Milvus: /tmp/milvus/etcdData/
-    dir: default.etcd
+  metaSubPath: meta # metaRootPath = rootPath + '/' + metaSubPath
```
Comment (haorenfsa) [inline configs/milvus.yaml:32]: The fields should be ordered, some fields should be put together, for example, the `xxxPath` fields

Comment (jiaoew1991) [reply configs/milvus.yaml:32]: I will try it

**Pull request merged**